src/appleseed/foundation/math/scalar.h

```diff
@@ -358,7 +358,11 @@ struct PowIntHelper
 {
     static T eval(const T x)
     {
-        return x * PowIntHelper<T, P - 1>::eval(x);
+        if(P % 2 == 0)
```

Comment:
Insert space between `if` and the opening parenthesis (as detailed in our [coding conventions](https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines#spacing)).